src/redux/rockets/rocketsSlice.js

```diff
@@ -1,4 +1,23 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const formattedBooks = (data) => data.map((rocket) => ({
+  id: rocket.id,
+  rocketname: rocket.rocket_name,
+  type: rocket.rocket_type,
+  image: rocket.flickr_images[0],
+}));
```

Comment:
Since this function is used to format data related to rockets, it would be more appropriate to rename it as formatRockets to ensure its clarity and consistency with the context.